### ISSUE TITLE
Confusing (useless) messages in ERRORR and COVR

### ISSUE THREAD

**Issue by luca-fiorito-11** [njoy/NJOY2016#85]
@whaeck As discussed, the messages coming out of errorr and covr that may be adapted.

When using Na-22 JEFF-3.3 evaluation useless messages are printed to the output file:

 ---message from rpxlc12---resonance parameter loop done 806.9s
---message from rpxlc12---sensitivity calculation continues 807.8s 
---message from rpxlc12---sensitivity calculation completed 809.6s 

Under certain conditions - e.g. Ho-166m JEFF-3.3 evaluation - COVR prints out incomplete messages.
In the example below, the MAT/MT pair is not updated after the 3rd and 4th message.

  ---message from matshd---processing of mat/mt 6729/ 3 vs. mat1/mt1 6729/103
 largest coefficient= -5.14345E+02 at index 186 224 
  ---message from matshd--- 62 coefficients > 1
 reset and continue. 
  ---message from matshd--- 146 coefficients > 2
 reset and continue 
  ---message from matshd--- 14 coefficients > 2
 reset and continue 
  ---message from matshd--- 32 coefficients > 2
 reset and continue

**Comment by whaeck** [comment]
Yes, the first messages you mention are essentially timing messages, which can be put in the output in a different way without using the standard call mess() mechanism.

For the last ones, some logic in covr prevents the MAT/MT to be printed if the correlation matrix contains only coefficients that are larger than 2. I will have to fix that.
